### ISSUE TITLE
Habits, Dailies, Todos, and Rewards all gone*

### ISSUE THREAD

**Issue by HHF** [HabitRPG/habitica#804]
*But somehow, one Daily survived.

I refreshed HabitRPG this morning and it was all just gone. I don't think I've created a new task for days, and I'm fairly sure the vast majority of these were weeks or even over a month old at this point, so I don't think this is a "refreshed during a server restart" issue. I don't know if this is relevant, but last night I was having trouble with my router, so multiple times I had to reconnect and refresh the page to get the last of my Dailies checked off before the connection died again. Still, that was only a few dailies versus three very large lists and two rewards that I created, and the one Daily that survived had already been checked off hours before my connection issues.

**Comment by lefnire** [comment]
@HHF can you try now?

**Comment by HHF** [comment]
Looks like everything's back, awesome. Everything's listed in a completely different order for some reason, but that's easy enough to fix. Thank you!

**Comment by lefnire** [comment]
Nail on the coffin for refLists. id-lists just get wiped all the freaking time, we'll definitely be moving to arrays https://github.com/lefnire/habitrpg/issues/634

**Comment by lefnire** [comment]
I know what happened. We're tracking unroll-me/derby#master and lefnire/racer#habitrpg. derby up'd the Racer dep to  0.3.15, but lefnire/racer was still at 0.3.14 - showing incompatible versions. So the good racer was installed top-level node_modules/, but never used - while the bad racer was installed within node_modules/derby. So all our old errors (which got fixed when we switched to SLaks/racer) came back for the last couple days. I'm forking unroll-me/derby to specify our racer dep to make sure this doesn't happen again.
